compassion_dashboard/README.rst

```diff
@@ -0,0 +1,27 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Currently, this module only provides a widget that ease the creation of line and bar graph within dashboard cards. Therefore, partner_communication does not depend on it yet but we could use it to generalize other dashboard related elements as we go. What do you think ?